spec/features/task_management_spec.rb

```diff
@@ -42,4 +51,9 @@ def force_reload_to_get_updated_submission_status
   def example_submission_file
     Rails.root.join('spec', 'fixtures', 'uploads', 'empty.xlsx')
   end
+
+  def mock_complete_submission_endpoint!
```

Comment:
perhaps this belongs with the other `ApiHelpers`?

Reply:
Maybe. It's specific to this one single test however.